scripts/license_checker.py

```diff
@@ -69,7 +69,7 @@
 ]
 
 # exclude file by file ending from license header check:
-EXCLUDE_ENDINGS = ["json", "pyc", "yaml", "yml", "md", "html", "xml"]
+EXCLUDE_ENDINGS = ["json", "pub", "pyc", "sec", "yaml", "yml", "md", "html", "xml"]
```

Comment:
About the `sec` ending: Should be ok for crypt4gh secret keys, though other secret keys do not necessarily have a file ending.